lib/info-extras.js

```diff
@@ -268,3 +268,54 @@ exports.cleanVideoDetails = videoDetails => {
     'videoDetails.shortDescription', 'videoDetails.description');
   return videoDetails;
 };
+
+/**
+ * Get storyboards info.
+ *
+ * @param {Object} info
+ * @returns {Array.<Object>}
+ */
+exports.getStoryboards = info => {
+  const parts = info.player_response.storyboards &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec.split('|');
+
+  if (!parts) return [];
+
+  const url = new URL(parts.shift());
+
+  return parts.map((part, i) => {
+    let [
+      width,
+      height,
+      count,
+      storyboardWidth,
+      storyboardHeight,
```

Comment:
what is `width` and `height` compared to `storyboardWidth`, `storyboardHeight`?

Reply:
Basically when you open the url you'll see a big picture comprised of small pictures representing different times of the video, the `width` and `height` are the dimensions of the small picture, and  `storyboardWidth` and `storyboardHeight` are the dimensions of the storyboard

Reply:
how about `rows` and `columns`?

Reply:
> how about `rows` and `columns`?

What will they represent? and will it replace other fields? or just an addition

Reply:
i mean in place of `storyboardWidth` and `storyboardHeight`. the current names can imply pixel width not images per side

Reply:
Oh I thought these fields would give pixel count, yes rows and columns would be much clearer

Reply:
and what do you think of instead of `width`/`height`, using `thumbnailWidth`/`thumbnailHeight`? i don't feel as strongly about this change as `rows`/`columns`, so would like to know what you think

Reply:
Hmmm, it would be more easier to read though, I think we should change it